larq/quantized_variable.py

```diff
@@ -7,7 +7,46 @@
 from larq import quantized_scope
 
 
-class QuantizedVariable(tf.Variable):
+class QuantizedVariableFactory(tf.Variable.__class__):
```

Comment:
`tf.Variable` is a `metaclass` itself, so we need to inherit from `tf.Variable.__class__` instead of `type` to prevent `metaclass` conflicts.